styles/abstracts/_variables.scss

```diff
@@ -1,6 +1,7 @@
 @import "initial-variables";
 @import "~bulma/sass/utilities/_all";
 @import "derived-variables";
+@import "theme";
```

Comment:
hmm.. no side effects?

Reply:
like what?

Reply:
Why is this line needed? 
I think there is a problem with the order of importing scss files, what do you think?

Reply:
since variables is already imported in most of the files, I thought it would be nice to import it here instead of single files? 
Nothing seems to break though